cs/unittest/TestTracing.cs

```diff
@@ -28,8 +28,9 @@ public void TestTraceListener()
             }
 
             var trace = string.Join("\n", messages);
-            Assert.AreEqual(16, messages.Count, $"Expected 16 lines. Found {messages.Count}. '{trace}'");
-            Assert.AreEqual("total feature number = 4", messages[15]);
+            var expected = 19;
```

Comment:
@lokitoth would this break something on the c# world?

Reply:
I do not believe so. While there may be someone depending on the specific exact output, it seems unreasonable to expect there to never be new trace messages as the version increments.

It does not seem reasonable for consumers of this API to assume that the contents of the trace logging are an API contract.